hw01_hello_otus/go.mod

```
@@ -1,3 +1,5 @@
-module github.com/fixme_my_friend/hw01_hello_otus
+module github.com/vadskev/hw01_hello_otus
```

```suggestion
module github.com/vadskev/otus-home-work/hw01_hello_otus
```
Наименование модуля должно содержать полный путь, включая название репозитория